samples/apps/copilot-chat-app/README.md

```diff
@@ -0,0 +1,77 @@
+# Copilot Chat Sample Application
+>! IMPORTANT  This learning sample is for educational purposes only and should not be used in any production use case.  It is intended to highlight concepts of Semantic Kernel and not any architectural / Security design practices to be used.
+
+## About the Copilot 
+The Copilot Chat sample allows you to build your own integrated large language model chatbot.  This is an enriched intelligence app, with multiple dynamic components including command messages, user intent, and memories.  The chat prompt and response will evolve as the conversation between the user and the application proceeds.  This chat experience is a chat skill containing multiple functions that work together to construct the final prompt for each exchange.
+
+
+![UI Sample](images/UI-Sample.png)
+
+## Dependencies:
+
+Before following these instructions, please ensure your development environment and these components are functional:
+1.	For this sample application, it is assumed that you already have VS Code installed as your IDE.
+2.	As the repository is stored on GitHub, Git should be installed as well.  
+3.	The sample application requires .NET 6.0 or greater.  
+4.	Node.js should be installed, 
+5.	Yarn will be required.
+
+
+## Running the Sample
+1.	You will need an Open AI Key or Azure Open AI Service Key for this sample.
+2.	You will need to register your application in the Azure Portal.  Follow the steps to register your app here.
+
+    1. Select Single-page application (SPA) as platform type, and the redirect URI will be `http://localhost:3000`
+    2.	Select `Accounts in any organizational directory and personal Microsoft Accounts` as supported account types for this sample.
+    3. Make a note of this Application (client) ID from the Azure Portal, we will make use of it later.
+3.	The sample uses two applications, a front-end web UI, and a back-end server.  First, let’s set up and verify the back-end server is running.
+
+    1. Navigate to `\samples\apps\copilot-chat-app\SKWebApi`
+    2.	Update `appsettings.json` with these settings:
+
+          *	Under the `“ServicePort”` block, make the following configuration changes to match your instance:
+
+            * `“AIService”: “AzureOpenAI”`, or whichever option is appropriate for your instance.
+            * `“DeploymentOrModelID”: “text-davinci-003”,` or whichever option is appropriate for your instance.  
+            * `“Endpoint”:` “Your Endpoint address, i.e. http://project.region.azure.com”
+            * `“Key”:` “Your azure endpoint key, i.e. a 36+ digit hex code” 
```

Comment:
We do not want anyone putting their key in this file. Recomment using "dotnet user-secrets" for running locally.


Open a command prompt and go to the `samples/apps/copilot-chat-app/SKWebApi` directory. Then run the following commands:
```
dotnet user-secrets set "CompletionConfig:Key" "YOUR KEY OPENAI or AZURE OPENAI KEY"
dotnet user-secrets set "EmbeddingConfig:Key" "YOUR KEY OPENAI or AZURE OPENAI KEY"
```

Reply:
These instuctions might make sense in the next section below ("Build the back-end server"). Maybe just make a reference here to setting the keys in that section.

Reply:
Updated